maxdrapa.md

```diff
@@ -0,0 +1,45 @@
+---
+name: "@maxdrapa"
+project: "skiLight"
+---
+
+# skiLight
+
+## Summary
+
+I am planning on integrating a custom lighting system into my ski helmet to act as headlights. I hope to be able to control this wirelessly through Bluetooth and even add some fun rainbow lighting. As a Vermonter, I'm always outside in the winter, and being able to have lights that won't take up any more space on my helmet is optimal. With this, my nighttime skiing and sledding will be improved greatly.
```

Comment:
Do you also need a bluetooth module for the microcontroller? I didn't see one in your budget.

Reply:
The microcontroller that I am getting comes with Bluetooth and wifi.

Reply:
Great!